### DISCUSSION TITLE
Data Access Segregation

### DISCUSSION THREAD

**Discussion by cyberpasta** [Security-Onion-Solutions/securityonion#2865] [Unsupported Versions] [answered]
Hellow Team. 
Is it possible with so2.3 to segregate data access per user or role in the SOC console? 
It is critical to protect some confidential logs from access to all the analysts.

**Comment by TOoSmOotH** [comment, marked as answer]
https://blog.securityonion.net/2020/12/security-onion-2-in-2020-and-2021.html

RBAC is a feature that we are planning on looking at this year.

**Reply by cyberpasta** [reply]
Many Thanks. That was helpful.
